### PR TITLE
Improve description of install options

### PR DESCRIPTION
**Pull request by DevSecNinja**
## Proposed change

Improve description of install options. I've also added some more context to the Intel NUC installation option.

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -115,9 +120,9 @@ If you would like to test next release before anyone else, you can install the b
 [pi4-32]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_rpi4-4.12.img.gz
 [pi4-64]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_rpi4-64-4.12.img.gz
 [tinker]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_tinker-4.12.img.gz
-[odroid-c2]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_odroid-c2-4.12.img.gz
-[odroid-n2]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_odroid-n2-4.12.img.gz
-[odroid-xu4]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_odroid-xu4-4.12.img.gz
+[ODROID-c2]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_odroid-c2-4.12.img.gz
+[ODROID-n2]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_odroid-n2-4.12.img.gz
+[ODROID-xu4]: https://github.com/home-assistant/operating-system/releases/download/4.12/hassos_odroid-xu4-4.12.img.gz
```
Comment (frenck) [inline source/hassio/installation.markdown]: These are link references, please keep them as is (lower case)

File: source/hassio/installation.markdown [hunk 2]
```diff
@@ -104,7 +109,7 @@ If you would like to test next release before anyone else, you can install the b
 3. Select _System_ tab from the _Supervisor_ menu, then select _Join Beta Channel_ under _Supervisor_, then select _Reload_.
 4. Select _Dashboard_ tab from the _Supervisor_ menu, and then select _Update_.
 
-
+[7-Zip]: https://www.7-zip.org/
```
Comment (frenck) [inline source/hassio/installation.markdown]:
```suggestion
[7-zip]: https://www.7-zip.org/
```

File: source/hassio/installation.markdown [hunk 3]
```diff
@@ -14,7 +14,7 @@ The following will take you through the steps required to install Home Assistant
      - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (recommended)
      - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
      - [Tinkerboard][tinker]
-     - [Odroid-C2][odroid-c2], [Odroid-N2][odroid-n2], [Odroid-XU4][odroid-xu4]
+     - [ODROID-C2][ODROID-c2], [ODROID-N2][ODROID-n2], [ODROID-XU4][ODROID-xu4]
```
Comment (frenck) [inline source/hassio/installation.markdown]: Please make the link references lower case

File: source/hassio/installation.markdown [hunk 4]
```diff
@@ -31,12 +31,17 @@ The following will take you through the steps required to install Home Assistant
 
 2. Install Home Assistant:
 
-   - Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
-   - Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
+   - **Raspberry Pi, Tinkerboard, or an ODROID**:
```
Comment (frenck) [inline source/hassio/installation.markdown]: The Intel NUC can also run from SD?

File: source/hassio/installation.markdown [hunk 5]
```diff
@@ -31,12 +31,17 @@ The following will take you through the steps required to install Home Assistant
...
+   Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
+   - **Virtual Appliance**:
```
Comment (frenck) [inline source/hassio/installation.markdown]: Emphasis should not be used in cases like this. This usually indicates headings are needed.

File: source/hassio/installation.markdown (hunk 4 above)
Comment (frenck) [inline source/hassio/installation.markdown]: Emphasis should not be used in cases like this. This usually indicates headings are needed.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/hassio/installation.markdown (hunk 4 above)
Comment (DevSecNinja) [reply source/hassio/installation.markdown]: Is that a supported scenario? Proposal:
````
Flash the downloaded image to your SSD/hard disk (recommended) or SD-card using...
````

Comment (frenck) [reply source/hassio/installation.markdown]: Nope, in general, the SSD isn't supported.

Comment (DevSecNinja) [reply source/hassio/installation.markdown]: Changed as discussed on Discord.

**Pull request closed without merging**